dgraph/cmd/zero/http.go

```diff
@@ -235,8 +237,27 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 func (st *state) pingResponse(w http.ResponseWriter, r *http.Request) {
 	x.AddCorsHeaders(w)
 
+	var healthAll []pb.HealthInfo
```

Comment:
Why is this a slice? It will just have 1 item.

Reply:
to make it consistent with zero's health response structure